### ISSUE TITLE
Root view window reference

### ISSUE THREAD

**Issue by kansaraprateek** [ephread/Instructions#268]
Hi,

I am trying to get access to new UIWindow reference created to show coaches. Is there a way to access that? 
My app has voice over accessibility feature support and I am unable to make that work due to multiple UIWindow. 
As every time a coach is shown the voice over accessibility works only with the views in the first window and not the one created for coaches.

**Comment by ephread** [comment]
Hello @kansaraprateek, sorry for the late response.

The window is not accessible, but it would be easy to expose it publicly. It's a good idea; Instructions doesn't handle accessibility very well. I have a batch of improvements to make to close other issues, so exposing the window will be part of it.

**Comment by ephread** [comment]
@kansaraprateek `CoachMarkController` now exposes `rootWindow`! Let me know if it solves your issue.
